include/envoy/router/rds.h

```diff
@@ -51,10 +51,10 @@ class RdsRouteConfigProvider : public RouteConfigProvider {
   virtual const std::string& routeConfigName() const PURE;
 
   /**
-   * @return const std::string& the name of the cluster the RdsRouteConfigProvider is issuing RDS
-   * requests to.
+   * @return const std::string& the configuration of the cluster the RdsRouteConfigProvider is
+   * issuing RDS requests to.
    */
-  virtual const std::string& clusterName() const PURE;
+  virtual const std::string& clusterConfig() const PURE;
```

Comment:
Rename to `configSource()`, the cluster aspect is no longer true in general, this could be a non-cluster destination `GrpcService`.